### ISSUE TITLE
Add Linux/Mac Executables

### ISSUE THREAD

**Issue by Pramad712** [Pramad712/Thornton-Shortest-Paths#4]
As of right now, only windows executables are in the releases page. It would help a lot if executables for other operating systems can be added.

**Comment by pradhan-ritik** [comment]
[TMS-Pathfinding-Linux.tar.gz](https://github.com/user-attachments/files/16882943/TMS-Pathfinding-Linux.tar.gz)
If you unzip this on linux it will give you the source code with the compiled version of the program and the assets. Not quite an executable but it works.

**Comment by Pramad712** [comment]
Is that the same one over here:
![image](https://github.com/user-attachments/assets/cd3b9a8e-ac7b-4842-8d2b-f281a4b9f2f2)

**Comment by Pramad712** [comment]
(that is a screenshot of the assets of the latest release)

**Comment by pradhan-ritik** [comment]
No. My version contains an executable that can be used to run the program plus everything in the one you have.

**Comment by Pramad712** [comment]
Adding that in!
